jiant/scripts/download_data/constants.py

```diff
@@ -6,7 +6,15 @@
     "multirc": f"https://dl.fbaipublicfiles.com/glue/superglue/data/v2/MultiRC.zip",
     "record": f"https://dl.fbaipublicfiles.com/glue/superglue/data/v2/ReCoRD.zip",
 }
-OTHER_DOWNLOAD_TASKS = {"abductive_nli", "swag", "qamr", "qasrl"}
+OTHER_DOWNLOAD_TASKS = {
+    "abductive_nli",
+    "swag",
+    "qamr",
+    "qasrl",
+    "ropes",
+    "newsqa",
+    "mrqa_natural_questions",
```

Comment:
Are there several versions of MRQA?

Reply:
MRQA is basically a repackaging/reformulation of several existing QA tasks, including Natural Questions. For the purposes of Clara's project, we're using the MRQA version of Natural Questions rather than the regular verison (which is much, much larger.)

Reply:
Can you add a supporting link to the `supported_tasks.md`? When I search MRQA natural questions, I am not sure which is the relevant result.

Reply:
Added.